README.md

```diff
@@ -1,7 +1,18 @@
 # FCL Contracts
 
-Cadence contracts used by the Flow Client Library (FCL).
-
+`FCLCrypto.cdc` and `FlowInteractionTemplateAudit.cdc` are contracts which can be used by the Flow Client Library (FCL).  
+  
+`FCLCrypto.cdc` allows to verify the signatures for an account and check that the combined weight of the account keys reaches signing power.  
+It provides the function `verifyUserSignatures` for user signatures and the function `verifyAccountProofSignatures` for account proof signatures.  
+These functions are accessible by the provided scripts `verifyUserSignatures.cdc` and `verifyAccountProofSignatures.cdc`.
+  
+In order to add [Interaction Templates](https://forum.onflow.org/t/flip-934-interaction-templates/3080) which shall be audited, use the `FlowInteractionTemplateAudit.cdc` contract and add the template by executing the transaction `add-audit.transaction.cdc` and provide the template id string. In order to revoke added audits, call the `revoke-audit.transaction.cdc` transaction.  
```

Comment:
Can you add a brief description of what the interaction template contract is and re word this paragraph to be a bit more clear?

Reply:
Thank you for the review. I worded it better.

Reply:
Thank you for the review. I worded it better.

Reply:
I now start with a short description of interaction templates, I think it should now be more clear.